.github/workflows/release.yml

```diff
@@ -0,0 +1,360 @@
+
+    - name: Build
+      run: cmake --build build --config ${{ matrix.config.name }} --target install
+
+    - name: Packaging - Copy Files
+      run: |
+        mkdir './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}'
+        cp -rH ./install/* './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}'
+        cp './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/share/doc/SFML/readme.md' './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}'
+        cp './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/share/doc/SFML/license.md' './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}'
+        rm -r './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/share'
+        cp ./changelog.md './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}'
+
+    - name: Packaging - Copy Frameworks for macOS
+      if: matrix.config.name == 'Frameworks' && matrix.type.name == 'Release' && runner.os == 'macOS'
+      run: |
+        mkdir './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/Frameworks'
+        mv './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/'*.framework './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/Frameworks'
+
+    - name: Packaging - Copy Example Files
+      if: matrix.config.name == 'Static' && matrix.type.name == 'Release'
+      run: |
+        mkdir 'SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/examples'
+        cp -r ./examples/* 'SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/examples'
+        rm './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/examples/CMakeLists.txt'
+        find './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/examples' -type f -name CMakeLists.txt -delete
+        rm -r './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/examples/android'
+        rm -r './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/examples/include'
+        rm -r './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/examples/vulkan/vulkan.h'
+        rm -r './SFML-${{ matrix.platform.name }}-${{ matrix.config.name }}-${{ matrix.type.name }}/examples/island/stb_perlin.h'
+
+    - name: Packaging - Prepare Example Files for Windows
```

Comment:
I don't think the value of shipping examples outweighs the effort and complexity required to do so. I expect that a rather small percentage of users of the prebuilt binaries take advantage of this. I'd rather we omit this from our CD pipeline so that our pipeline remains easier to maintain and our install packages are lighter.

Reply:
I do still think it has value for:
- Checking that SFML "works" on the user's PC
- Easily accessible SFML showcase

binary1248 also had the idea, that I kind of like, to ship the examples separately